### PR TITLE
Quiz page

### PR DESCRIPTION
**Pull request by dipen2000**
I created this quiz page for the quiz app so please review this PR and give me your feedback.

### CODE REVIEW

File: CSS/styles.css [hunk 1]
```diff
@@ -42,14 +46,45 @@
...
+
 .flex-col{
     display : flex;
     justify-content: center;
     align-items : center;
 }
 
 .container{
-    border : 3px solid red;
+    /* border : 3px solid red; */
```
Comment (pnchinmay) [inline CSS/styles.css:87]: remove redundant comments

**Review by pnchinmay — APPROVED** [review]

**Comment by shruthipatil11** [comment]
look great

File: CSS/styles.css [hunk 2]
```diff
@@ -58,14 +93,15 @@
 header{
     font-family: var(--heading-font);
     font-size : 3rem;
+    font-weight : bold;
```
Comment (dev-ashishsethi) [inline CSS/styles.css:96]: could have used <strong> tag

File: CSS/styles.css [hunk 3]
```diff
@@ -77,17 +113,64 @@ ol{
...
+}
 
+.single-category-container{
+    /* border : 3px solid green; */
+    display : flex;
+    flex-direction: column;
+    padding : 1.2rem;
+    width : 35rem;
+    gap : 1rem;
+    border-radius: 0.8rem;
```
Comment (dev-ashishsethi) [inline CSS/styles.css:138]: use pixels when less than 1rem

**Review by dev-ashishsethi — APPROVED** [review]

File: CSS/styles.css [hunk 4]
```diff
@@ -58,14 +93,15 @@
 header{
     font-family: var(--heading-font);
     font-size : 3rem;
+    font-weight : bold;
 }
 
 main{
-    border : 3px solid red;
+    /* border : 3px solid red; */
```
Comment (j24m) [inline CSS/styles.css:100]: Remove the commented code.

File: CSS/styles.css [hunk 5]
```diff
@@ -58,14 +93,15 @@
...
 }
 
 .main-container{
-    border : 3px solid green;
+    /* border : 3px solid green; */
```
Comment (j24m) [inline CSS/styles.css:104]: here too.

File: CSS/styles.css [hunk 6]
```diff
@@ -77,17 +113,64 @@ ol{
 }
 
 .rules-container{
-    border : 3px solid yellow;
+    /* border : 3px solid yellow; */
```
Comment (j24m) [inline CSS/styles.css:116]: here.

File: CSS/styles.css [hunk 7]
```diff
@@ -77,17 +113,64 @@ ol{
...
     display : flex;
     flex-direction: column;
     gap : 3rem;
     width : 60%;
 }
 
+.categories-flex-container{
+    /* border : 3px solid yellow; */
```
Comment (j24m) [inline CSS/styles.css:124]: here.

File: CSS/styles.css [hunk 8]
```diff
@@ -77,17 +113,64 @@ ol{
...
+    display : flex;
+    flex-direction: row;
+    justify-content : center;
+    gap : 2rem;
+}
 
+.single-category-container{
+    /* border : 3px solid green; */
```
Comment (j24m) [inline CSS/styles.css:132]: here

**Review by j24m — APPROVED** [review]
Hello, In this file too I see that you haven't removed the commented code, please do that before merging , rest everything looks good:)

**Comment by shruthipatil11** [comment]
looks good to merge

File: CSS/styles.css (hunk 2 above)
Comment (dipen2000) [reply CSS/styles.css:96]: ok got it, <strong></strong>

File: CSS/styles.css [hunk 9]
```diff
@@ -77,17 +113,64 @@ ol{
...
 
-
-
+.question-container{
+    /* border : 3px solid blue; */
+    width : 60%;
+    max-width: 60rem;
+    padding : 1rem;
+    border-radius: 1rem;
+    background-color : var(--light-test-color);
```
Comment (rahulyadav139) [inline CSS/styles.css:167]: here you used "--light-test-color" it should be named any understandable name. I am not this is your final color naming or not. Other than that everything is clean and simple. Great work

**Review by rahulyadav139 — APPROVED** [review]

**Pull request merged**